frontend/package.json

```diff
@@ -5,6 +5,7 @@
   "author": "Julien_P",
   "private": true,
   "scripts": {
+    "analyze": "nuxt build --analyze",
```

Comment:
I think this is enough to trigger `analyze` you don't need to add the build config, it's one or the other.

Reply:
fair enough :)